sdk/Pulumi.Automation/Commands/IPulumiCmd.cs

```diff
@@ -5,11 +5,14 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Pulumi.Automation.Events;
+using Semver;
 
 namespace Pulumi.Automation.Commands
 {
-    internal interface IPulumiCmd
```

Comment:
The dotnet SDK already uses a class to encapsulate running pulumi command, but it wasn't public. Now that users can supply their own instances to LocalWorkspace, we make this a public.